tests/test_MigrateData.py

```diff
@@ -0,0 +1,1072 @@
+import pytest
+
+import KoalaBot
+from utils.KoalaDBManager import KoalaDBManager
+from utils.MigrateData import MigrateData
+
+database_manager = KoalaDBManager(KoalaBot.DATABASE_PATH, KoalaBot.DB_KEY, KoalaBot.CONFIG_DIR)
+database_manager.create_base_tables()
+migrate_database = MigrateData(database_manager)
+
+TABLE_NAME = "{TABLE_NAME}"
+DROP_TABLE_SQL = f"DROP TABLE {TABLE_NAME}"
+
+
+def create_base_tables():
```

Comment:
docstrings for why this function is required please, also is this similar to line 8?